src/exts/message.py

```diff
@@ -64,6 +65,10 @@ async def help_thread_creation_modal(
             else content,
             description="\n".join(content.split("\n")[1:]),
             color=0xFEE75C,
+            footer=interactions.EmbedFooter(
+                text="Please create a thread in #help to ask questions!"
+            ),
+            timestamp=datetime.now(timezone.utc),
```

Comment:
Use the target message's `timestamp` field for this. This will only reflect creation of the thread and not when the question was originally asked